src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,57 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile = readFromFile(fromFileName);
+        String report = createReport(dataFromFile);
+        writeToFile(toFileName, report);
+    }
+
+    private String readFromFile(String fromFileName) {
+        StringBuilder firstFileData = new StringBuilder();
+        try (BufferedReader br = new BufferedReader(new FileReader(fromFileName))) {
+            String value = br.readLine();
+            while (value != null) {
+                firstFileData.append(value).append(System.lineSeparator());
+                value = br.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
+        }
+        return firstFileData.toString();
+    }
+
+    private String createReport(String dataFromFile) {
+        StringBuilder report = new StringBuilder();
+        int supply = 0;
+        int buy = 0;
+        for (String str : dataFromFile.split(System.lineSeparator())) {
+            String[] splitted = str.split(",");
+            if (splitted[0].equals("buy")) {
```

Comment:
![image](https://user-images.githubusercontent.com/12864128/188132261-ede4b80c-ee65-43cf-ae24-b9933aacb607.png)

Create constant for indexes your splitted array. Like splitted[BUY_INDEX].equals("buy")

Reply:
@kozhukhovsky I think it's better to use ACTION_INDEX, than BUY_INDEX, cuz zero index has 2 options (buy/supply). But this is just my opinion, please correct me if I think wrong